src/main/java/services/CompanyService.java

```diff
@@ -0,0 +1,33 @@
+package services;
+
+import entities.Company;
+
+import javax.persistence.EntityManager;
+import javax.persistence.Persistence;
+
+public class CompanyService {
```

Comment:
Please write a contract on the interface. And this class should implement this contract.